autotest/autotest.cpp

```diff
@@ -2006,6 +2016,42 @@ bool tAutotest::step_dumpPackets(const YAML::Node& yamlStep,
 	return true;
 }
 
+bool tAutotest::step_bird(const YAML::Node& yamlStep,
+                          const std::string& path)
+{
+	std::string confPath;
+	for (const auto& yamlIter : yamlStep)
+	{
+		confPath = yamlIter.as<std::string>();
+	}
```

Comment:
why do we reinitializing that string in a loop?
Maybe just `std::string confPath = yamlStep[0].as<std::string>();` is enough?